documentation/src/main/asciidoc/reference/search-dsl-predicate.asciidoc

```diff
@@ -1445,6 +1445,27 @@ or for the whole predicate with a call to `.boost(...)`
 after `.circle(...)`/`.boundingBox(...)`/`.polygon(...)`.
 
 
+[[search-dsl-predicate-not]]
+== `not`: negating another predicate
+
+The `not` predicate simply negates another predicate.
```

Comment:
Let's try to add something compared to the title :)

The documentation for other predicates uses phrasing similar to this:

```suggestion
The `not` predicate matches documents that are not matched by a given predicate.
```